[docs] More API fixes (#3640)
huggingface/diffusers: docs/source/en/api/diffusion_pipeline.mdx

# Pipelines

The [`DiffusionPipeline`] is the easiest way to load any pretrained diffusion pipeline from the [Hub](https://huggingface.co/models?library=diffusers) and use it for inference.

<Tip>
	
You shouldn't use the [`DiffusionPipeline`] class for training or finetuning a diffusion model. Individual 
components (for example, [`UNetModel`] and [`UNetConditionModel`]) of diffusion pipelines are usually trained individually, so we suggest directly working with instead.

</Tip>

> The pipeline type (for example [`StableDiffusionPipeline`]) of any diffusion pipeline loaded with [`~DiffusionPipeline.from_pretrained`] is automatically 
> detected and consequently loads each component of the 
> pipeline and passes them into the `__init__` function of the pipeline.

Reviewer: ```suggestion
The pipeline type (for example [`StableDiffusionPipeline`]) of any diffusion pipeline loaded with [`~DiffusionPipeline.from_pretrained`] is automatically 
detected and pipeline components are loaded and passed to the `__init__` function of the pipeline.
```

# Pipelines

The [`DiffusionPipeline`] is the easiest way to load any pretrained diffusion pipeline from the [Hub](https://huggingface.co/models?library=diffusers) and use it for inference.

<Tip>
	

> 	You shouldn't use the [`DiffusionPipeline`] class for training or finetuning a diffusion model. Individual 

Reviewer 2: ```suggestion
	You shouldn't use the [`DiffusionPipeline`] class for training or finetuning a diffusion 
model. Individual 
	components ([`UNet1DModel`], [`UNet2DConditionModel`], for example) of diffusion pipelines are usually trained individually, so we suggest directly working with them instead.
```

Reviewer 2: The suggestion might be broken. Proceed with caution.

	- device
	- to
	- components

## ImagePipelineOutput
By default diffusion pipelines return an object of class

[[autodoc]] pipelines.ImagePipelineOutput

## AudioPipelineOutput
By default diffusion pipelines return an object of class

[[autodoc]] pipelines.AudioPipelineOutput

## ImageTextPipelineOutput
By default diffusion pipelines return an object of class

> [[autodoc]] ImageTextPipelineOutput

Reviewer 2: Where are these going?

Reviewer 2: Ah nevermind.
